test/base/NumericLimits.test.cc

```diff
@@ -15,31 +15,59 @@
 using namespace celeritas_test;
 
 //---------------------------------------------------------------------------//
-// TEST HARNESS
+// REAL TYPES
 //---------------------------------------------------------------------------//
 
 template<class T>
-class NumericLimitsTest : public celeritas::Test
+class RealNumericLimitsTest : public celeritas::Test
 {
-  protected:
-    void SetUp() override {}
 };
-
 using RealTypes = ::testing::Types<float, double>;
-TYPED_TEST_SUITE(NumericLimitsTest, RealTypes);
+TYPED_TEST_SUITE(RealNumericLimitsTest, RealTypes, );
+
+TYPED_TEST(RealNumericLimitsTest, host)
+{
+    using std_limits_t   = std::numeric_limits<TypeParam>;
+    using celer_limits_t = celeritas::numeric_limits<TypeParam>;
+
+    EXPECT_EQ(std_limits_t::epsilon(), celer_limits_t::epsilon());
+    EXPECT_EQ(std_limits_t::infinity(), celer_limits_t::infinity());
+    EXPECT_EQ(std_limits_t::max(), celer_limits_t::max());
+    EXPECT_TRUE(std::isnan(celer_limits_t::quiet_NaN()));
+}
+
+#if CELERITAS_USE_CUDA
```

Comment:
Which part of this test is run on the device?

Reply:
`nl_test()` launches the kernel in NumericLimits.test.cu